package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "react-sign-up-with-tests",
-  "version": "0.1.0",
-  "private": true,
+  "version": "0.2.0",
+  "license": "MIT",
   "dependencies": {
-    "@testing-library/dom": "10.4.0",
-    "@testing-library/jest-dom": "6.4.8",
-    "t @testing-library/react": "16.0.0",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.4.8",
+    "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "14.5.2",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-scripts": "5.0.1",
-    "web-vitals": "4.2.3"
+    "react": "^18.3.1",
```

Comment:
can we remove ^ from all over

Reply:
Why?
We want the upgraded minor version updates when we push the next set of dependencies upgrade. Don't we?